index.js

```diff
@@ -86,12 +87,20 @@ function assembleStyles() {
 			};
 
 			group[styleName] = styles[styleName];
+
+			codes.set(style[0], style[1]);
 		}
 
 		Object.defineProperty(styles, groupName, {
 			value: group,
 			enumerable: false
 		});
+
+		Object.defineProperty(styles, 'codes', {
+			value: codes,
+			enumerable: false,
+			configurable: true
+		});
```

Comment:
Hm yeah. For some reason it `threw` yesterday without it, but worked now so must've done something else wrong.